Dockerfile

```
@@ -8,25 +8,36 @@ RUN --mount=source=.,target=/mnt,type=bind \
                                                vim python3-all pciutils binutils tcpdump \
                                                rsyslog iptables ssl-cert openssl ca-certificates \
                                                curl nmap hydra hping3 iperf3 d-itg socat \
- && apt install -y /mnt/misc/shellinabox_2.21.1_amd64.deb \
- && echo "root:hackinsdn" | chpasswd \
+ && apt install -y /mnt/misc/shellinabox_2.21.1_amd64.deb
+
+RUN rm -rf /var/lib/apt/lists/*
+
+RUN --mount=source=.,target=/mnt,type=bind \
+    cp -vr /mnt/conf/* / \
+ && curl -L --insecure https://trex-tgn.cisco.com/trex/release/v3.05.tar.gz | tar -xz -C /opt/trex
+
+RUN echo "root:hackinsdn" | chpasswd \
  && sed -i 's/#ListenAddress 0.0.0.0/ListenAddress 127.0.0.1/g' /etc/ssh/sshd_config \
- && sed -i 's/#PermitRootLogin prohibit-password/PermitRootLogin yes/g' /etc/ssh/sshd_config \
- && cp -r /mnt/src /var/www/html/secflood \
- && cp -vr /mnt/conf/* / \
- && curl -L --insecure https://trex-tgn.cisco.com/trex/release/v3.05.tar.gz | tar -xz -C /opt/trex \
- && install --mode 0755 --owner root /mnt/scripts/trex /usr/local/bin/ \
- && mkdir -p /var/lib/sqlite \
+ && sed -i 's/#PermitRootLogin prohibit-password/PermitRootLogin yes/g' /etc/ssh/sshd_config
+
+RUN --mount=source=.,target=/mnt,type=bind \
+    cp -r /mnt/src /var/www/html/secflood \
```

Here we can add the additional configs removed on the previous suggestion:

```suggestion
RUN --mount=source=.,target=/mnt,type=bind \
    cp -r /mnt/src /var/www/html/secflood \
 && cp -vr /mnt/conf/* / \
```